resources/translations/en.edn

```diff
@@ -109,7 +109,7 @@
                    :license-name "License name"
                    :licenses "Licenses"
                    :modifier "Modifier"
-                   :more-info "More info URL (optional, defaults to resource URN)"
+                   :more-info "More info URL (optional, defaults to resource URN or EGA link)"
```

Comment:
Should we add this text if the EGA stuff is behind a feature flag? Perhaps modify the texts only when `:enable-ega` defaults to `true`?

Reply:
I guess that added complexity is not worthwhile for this. I did however want to add the `:enable-ega` flag here as in theory someone else's resource ids could start `"EGAD"`. We could perhaps remove this complexity from the localizations altogether and have the manual explain the automatic parts.

Reply:
So do you want to leave the description in this localization? It feels a bit weird since I imagine almost none of our current users would have `:enable-ega true` in the near future.

Reply:
Let's remove it altogether then? For URN too?

Reply:
LBR could decide to put the URN as an extra translation for example, for others I think manual is the correct place if any.

Reply:
Fine by me!